stellar-multitenant/pkg/tenant/manager_test.go

```diff
@@ -363,31 +363,31 @@ func Test_Manager_GetTenantByIDOrName(t *testing.T) {
 	require.NoError(t, err)
 
 	t.Run("gets tenant by ID successfully", func(t *testing.T) {
-		tntDB, err := m.GetTenantByIDOrName(ctx, tnt1.ID)
-		require.NoError(t, err)
+		tntDB, dbErr := m.GetTenantByIDOrName(ctx, tnt1.ID)
+		require.NoError(t, dbErr)
 		assert.Equal(t, tnt1, tntDB)
 	})
 
 	t.Run("gets tenant by name successfully", func(t *testing.T) {
-		tntDB, err := m.GetTenantByIDOrName(ctx, tnt2.Name)
-		require.NoError(t, err)
+		tntDB, dbErr := m.GetTenantByIDOrName(ctx, tnt2.Name)
+		require.NoError(t, dbErr)
 		assert.Equal(t, tnt2, tntDB)
 	})
 
 	t.Run("returns error when tenant is deactivated", func(t *testing.T) {
 		deactivateTenant(t, ctx, m, tnt2)
-		tntDB, err := m.GetTenantByIDOrName(ctx, tnt2.ID)
-		assert.ErrorIs(t, err, ErrTenantDoesNotExist)
+		tntDB, dbErr := m.GetTenantByIDOrName(ctx, tnt2.ID)
+		assert.ErrorIs(t, dbErr, ErrTenantDoesNotExist)
 		assert.Nil(t, tntDB)
 
-		tntDB, err = m.GetTenantByIDOrName(ctx, tnt2.Name)
-		assert.ErrorIs(t, err, ErrTenantDoesNotExist)
+		tntDB, dbErr = m.GetTenantByIDOrName(ctx, tnt2.Name)
+		assert.ErrorIs(t, dbErr, ErrTenantDoesNotExist)
 		assert.Nil(t, tntDB)
 	})
 
 	t.Run("returns error when tenant is not found", func(t *testing.T) {
-		tntDB, err := m.GetTenantByIDOrName(ctx, "unknown")
-		assert.ErrorIs(t, err, ErrTenantDoesNotExist)
+		tntDB, dbErr := m.GetTenantByIDOrName(ctx, "unknown")
+		assert.ErrorIs(t, dbErr, ErrTenantDoesNotExist)
 		assert.Nil(t, tntDB)
 	})
 }
```

Comment:
🤔  Were these changes needed due to shadowing issues, or did you change it because you preferred this new name for the error variable?

We don't usually give custom names to error variables unless they are a specific error type, or to prevent variable shadowing.

Reply:
yes exactly, I had to change them due to shadowing issues that the linter didn't like!